testcafe/tests/drive/search.js

```diff
@@ -0,0 +1,108 @@
+import { driveUser } from '../helpers/roles'
+//import { Role } from 'testcafe'
```

Comment:
Please remove commented lines if they aren't used.

Reply:
I removed `//import { Role } from 'testcafe'` but left `//import ViewerDrivePage from '../pages/drive-viewer/drive-viewer-model'` and `//const viewerDrivePage = new ViewerDrivePage()` as theyĺl be needed when un-commenting the part about the viewer bug